cmd/provisioning/provisioning.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2024 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package provisioning
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewSubCmd() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:              "provisioning",
```

Comment:
"provisioning" is a long command to type. Would it make sense to define a shorter alias (while leaving "provisioning" as the canonical form), such as "prov"? (cf. kubectl service == kubectl svc)

Reply:
fsoc tep lookup your-vanity-url.appdynamics.com